stable/labelsmanager-controller/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for labelsmanager-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+controller:
+  # controller.production -- The log level of the controller. enable for json logs
+  production: true
+  # controller.leaderElection -- Enable leader election for running multiple controller pods
+  leaderElection: false
+  # controller.metricAddr -- Address to serve prometheus metrics on. "0" is disabled.
+  metricAddr: "0"
+
+  # controller.overwrite -- Overwrite any similar label on the pod.
+  overwrite: false
+
+image:
+  repository: thomasnyambati/labelsmanager-controller
+  pullPolicy: IfNotPresent
+  tag: ""
```

Comment:
Can you create a tag and put it here?

Reply:
I am using the appVersion as default value, we can add this incase we use AWS registry

Reply:
Ah I see. Nice. Just leave it like this then. I see it's the default behaviour in Helm now.